src/utils/common.h

```diff
@@ -439,6 +439,8 @@ typedef u64 __bitwise le64;
 #endif /* __GNUC__ */
 #endif /* __must_check */
 
+#include "verifyssid.h"
```

Comment:
if we include this from common.h, every file including it will have to parse verifyssid.h even though its only needed in one file, thus making the build unnecessarily slower